### PR TITLE
MWPW-143655

### PR DESCRIPTION
**Pull request by Blainegunn**
**CODEOWNERS for Main**
This only contains T1 Consumers / Sign-off teams on the `main` branch now. The Original `CODEOWNERS` will remain as is on `stage`.

Resolves: [MWPW-143655](MWPW-143655)
Linked to: https://github.com/adobecom/milo/pull/1950
**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://mwpw-143655--milo--adobecom.hlx.page/?martech=off

### CODE REVIEW

**Comment by Blainegunn** [comment]
Not sure the error are correct. 
All teams are [here](https://github.com/orgs/adobecom/teams?query=sot)

**Comment by honstar** [comment]
> Not sure the error are correct. All teams are [here](https://github.com/orgs/adobecom/teams?query=sot)

None of the `sot` teams have write access to the Milo repository which is a prerequisite.

**Review by mokimo — CHANGES_REQUESTED** [review]
Requesting changes until [this question about merge conflicts](https://github.com/adobecom/milo/pull/1950#pullrequestreview-1913532657) is answered // proven on a fork that this is not an issue.

**Pull request closed without merging**
